### PR TITLE
waldschadenkarte: tooltip, legends and translations

### PR DESCRIPTION
**Pull request by simonevonheute**
can you check everything concerning tooltip, legends and translations for this part? 

Test link: https://mf-geoadmin3.dev.bgdi.ch/ltdai/src/?topic=ech&lang=fr&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.vbs.waldschadenkarte&X=247181.57&Y=612225.50&zoom=10&catalogNodes=687,752

### CODE REVIEW

**Comment by gjn** [comment]
jenkins test

**Comment by loicgasser** [comment]
Looks good thank you very much.

Could you deploy a test link please?

```
git checkout dai_tooltip
make deploybranch
sudo /etc/init.d/apache2 restart
```

And let me know when it's done, please?

**Comment by loicgasser** [comment]
Also is this supposed to be deployed this week?
If yes, please put a tag next deploy under `Milestone` on the right.

**Comment by loicgasser** [comment]
Code is ok.

In DB there is no primary key on bgdi_id, please add it.

There are 2 geometrics indexes on the table, please get rid of one.

Could you please enforce nullable? Is column nullable or not?

Have a look at my comments in https://github.com/geoadmin/mf-chsdi3/pull/2124

We usually add btree indices on queryable attributes, could you do that?

**Comment by simonevonheute** [comment]
translation updated

**Comment by simonevonheute** [comment]
testlink: https://mf-geoadmin3.dev.bgdi.ch/ltdai/src/?topic=ech&lang=fr&bgLayer=ch.swisstopo.pixelkarte-grau&layers=ch.vbs.waldschadenkarte&X=223837.50&Y=628275.00&zoom=2&catalogNodes=687

**Comment by loicgasser** [comment]
Nice in french the legend content has unexpected bullets.
And it's versement not verse-ment.

To be corrected in GeoCat and reimported.

**Comment by simonevonheute** [comment]
ready to merge

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -5786,6 +5786,27 @@ msgstr "Territorial regions"
 msgid "ch.vbs.url"
 msgstr "http://www.vbs.admin.ch/internet/vbs/en/home.html"
 
+msgid "ch.vbs.waldschadenkarte"
+msgstr "Waldschadenkarte - Projektil"
```
Comment (oterral) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: No english translations?

Comment (simonevonheute) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: no, for english german has to be used if I understood right : 
http://www.geocat.ch/geonetwork/srv/eng/catalog.search#/metadata/2ff2299e-55b5-4ec7-bb92-fbb06e7ee422

**Comment by AFoletti** [comment]
the layer does not work from your testlink. Could you please rebase the branch and deploy it?
Is there a sphinx index?

**Comment by simonevonheute** [comment]
https://mf-geoadmin3.dev.bgdi.ch/src/?api_url=dai_tooltip&topic=ech&lang=fr&bgLayer=ch.swisstopo.pixelkarte-grau&layers=ch.vbs.waldschadenkarte&X=166196.75&Y=623019.50&zoom=12&catalogNodes=687

**Comment by AFoletti** [comment]
jenkins test

File: chsdi/models/vector/vbs.py [hunk 2]
```diff
@@ -141,3 +141,22 @@ class Logistikraeume_armeelogistikcenter(Base, Vector):
...
+
+class Waldschaden(Base, Vector):
+    __tablename__ = 'projektil'
+    __table_args__ = ({'schema': 'wascha', 'autoload': False})
+    __template__ = 'templates/htmlpopup/waldschadenkarte.mako'
+    __bodId__ = 'ch.vbs.waldschadenkarte'
+    __queryable_attributes__ = ['lauf_nr', 'jahr_schad', 'gde_name', 'lokalname', 'x_koord', 'y_koord']
+    __label__ = 'lauf_nr'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    the_geom = Column(Geometry3D)
```
Comment (AFoletti) [inline chsdi/models/vector/vbs.py]: Geometry3D ?

**Comment by AFoletti** [comment]
put a small comment in the model. Apart from that looks good to me and ready to be merged

**Comment by simonevonheute** [comment]
squashed all my pushs for waldschadenkarte

**Comment by loicgasser** [comment]
LGTM

**Pull request merged**